Samples/Samples/Samples/Views/EffectsEntryDisableAutoCorrectPage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace FormsCommunityToolkit.Samples.Views
+{
+	[XamlCompilation(XamlCompilationOptions.Compile)]
```

Comment:
No reason at all! I keep forgetting that VS2017 auto adds this to the pages when you select Add new content page...
Will adjust this :)